test/spec/modules/madvertiseBidAdapter_spec.js

```diff
@@ -70,26 +72,70 @@ describe('madvertise adapater', () => {
   });
 
   describe('Test build request', () => {
-    it('minimum request', () => {
-      let bid = [{
-        bidder: 'madvertise',
-        sizes: [[728, 90], [300, 100]],
-        bidId: '51ef8751f9aead',
-        adUnitCode: 'div-gpt-ad-1460505748561-0',
-        transactionId: 'd7b773de-ceaa-484d-89ca-d9f51b8d61ec',
-        auctionId: '18fd8b8b0bd757',
-        bidderRequestId: '418b37f85e772c',
-        params: {
-          s: 'test',
+    beforeEach(function () {
+      let mockConfig = {
+        consentManagement: {
+          cmpApi: 'IAB',
+          timeout: 1111,
+          allowAuctionWithoutConsent: 'cancel'
```

Comment:
Related to above, this should be a `boolean` value (`false` in this case) not the string `cancel`.  Passing in a string here actually causes the config to the opposite value (ie `true`) which allows the auction to proceed without consent.